decidim-budgets/app/models/decidim/budgets/project.rb

```diff
@@ -36,6 +37,11 @@ def comments_have_votes?
       def confirmed_orders_count
         orders.finished.count
       end
+
+      # Public: Overrides the `notifiable?` Notifiable concern method.
+      def notifiable?(_context)
+        false
```

Comment:
So is this notifiable or not? I mean, I feel it's weird having the `Notifiable` module included, but it not actually being notifiable. Maybe a better naming would solve this, but the current state feels very confusing to me :/